Tasks/AppCenterDistributeV1/task.json

```diff
@@ -12,10 +12,10 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 138,
+        "Minor": 139,
         "Patch": 0
     },
-    "releaseNotes": "Provide build id to distribution API for additional context.",
+    "releaseNotes": "Add support for distribution to stores.",
```

Comment:
"Release notes" is a text that the app's contributor adds to describe (lack a better word) the new release. 
`Add support for distribution to stores.` does not explain what "Release notes" is. 

@pniko / @DennisPan  any suggestion for a string that represents Release Notes.

Reply:
Ok, synced offline.